Header header

uint32 info
KF[] KeyFrames
MP[] MapPoints
